workers/generic-worker/main.go

```diff
@@ -595,6 +595,7 @@ func (task *TaskRun) validatePayload() *CommandExecutionError {
 		panic(err)
 	}
 	if _, exists := payload["image"]; exists {
+		task.Info("Docker Worker payload detected. Converting to a Generic Worker payload using d2g.")
```

Comment:
I guess we don't want to dump resulting converted payload, as it would be visible in live log anyways?